job_runner.py

```diff
@@ -0,0 +1,97 @@
+import os
+from jsonschema import validate, ValidationError
+import yaml
+from test_runner import TestRunner
+from compliance_suite.exceptions.ComplianceException import ComplianceException
+from compliance_suite.functions.colored_console import print_blue, print_red, print_green, print_yellow
+
+
+class JobRunner():
+
+    def __init__(self, tags):
+        self.path = os.getcwd()
+        self.tags = tags
+        self.test_count = 0
+        self.result = {             # To store the count of passed/failed/skipped tests
```

Comment:
I think it'd be better to provide a method that calculates the numeric results on the fly by processing `.test_status`. No need to store and update this information separately